tracer/test/test-applications/integrations/LogsInjection.ILogger/LogsInjection.ILogger.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFrameworks>net48;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1;net5.0;net6.0;net7.0;net8.0;net9.0</TargetFrameworks>
+    <TargetFrameworks>net48;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1;net5.0;net6.0;net7.0;net8.0;net9.0;net10.0</TargetFrameworks>
```

Comment:
There are so many of these. Can't we just remove this line so it picked up the shared one in `tracer/test/test-applications/Directory.Build.props`?